cmd/gtc/main.go

```diff
@@ -27,18 +29,22 @@ func main() {
 	fset := pflag.NewFlagSet("gtc", pflag.ExitOnError)
 	fset.BoolVarP(&Help, "help", "h", false, "show help message")
 	fset.BoolVarP(&Verbose, "verbose", "v", true, "verbose output")
+	fset.BoolVar(&Silence, "silence", true, "silence log output")
 	fset.Parse(os.Args)
 
 	show, err := fset.GetBool("help")
 	if err != nil {
 		log.Fatal(err)
 	}
+
 	if show {
 		fmt.Println(Usage)
 		os.Exit(84)
 	}
-
 	args := os.Args
+	if Silence {
+		args = pop(args, "--silence")
```

Comment:
Not a big deal, more a thought. Pop is often used in languages to get the last element whereas it looks like this function is more removeItem.